src/Core/FeatureFlag/FeatureFlagsModifier.php

```diff
@@ -0,0 +1,139 @@
+ * and handles payloads from the Back Office page, replicating the changes on the data model.
+ */
+class FeatureFlagsModifier implements DataConfigurationInterface
+{
+    /** @var EntityManagerInterface */
+    private $doctrineEntityManager;
+
+    /** @var TranslatorInterface */
+    private $translator;
+
+    /**
+     * @param EntityManagerInterface $doctrineEntityManager
+     * @param TranslatorInterface $translator
+     */
+    public function __construct(EntityManagerInterface $doctrineEntityManager, TranslatorInterface $translator)
+    {
+        $this->doctrineEntityManager = $doctrineEntityManager;
+        $this->translator = $translator;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfiguration(): array
+    {
+        $configuration = [];
+
+        foreach ($this->getAllFeatureFlags() as $id => $featureFlag) {
+            $configuration[$featureFlag->getName()] = $featureFlag->getState();
+        }
+
+        return $configuration;
+    }
+
+    /**
+     * @return array<int, FeatureFlag> $allFlags
+     */
+    public function getAllFeatureFlags(): array
+    {
+        return $this->doctrineEntityManager->getRepository('PrestaShopBundle:FeatureFlag')->findAll();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function updateConfiguration(array $configuration): array
+    {
+        /** @var array<string, boolean> $configuration */
+        foreach ($configuration as $flagName => $flagState) {
+            $this->validateConfigurationRowType($flagName, $flagState);
+
+            $featureFlag = $this->getOneFeatureFlagByName($flagName);
+
+            if ($flagState) {
+                $featureFlag->enable();
+            } else {
+                $featureFlag->disable();
+            }
+        }
+
+        $this->doctrineEntityManager->flush();
+
+        return [];
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function validateConfiguration(array $configuration): bool
+    {
+        /** @var array<string, boolean> $configuration */
+        foreach ($configuration as $flagName => $flagState) {
+            $this->validateConfigurationRowType($flagName, $flagState);
+
+            if (null === $this->getOneFeatureFlagByName($flagName)) {
+                return false;
+            }
+        }
+
+        return true;
+    }
+
+    /**
+     * @param string $featureFlagName
+     *
+     * @return FeatureFlag|null return null if feature flag cannot be found
+     */
+    private function getOneFeatureFlagByName(string $featureFlagName)
+    {
+        return $this->doctrineEntityManager->getRepository('PrestaShopBundle:FeatureFlag')->findOneBy(['name' => $featureFlagName]);
+    }
+
+    /**
+     * @param string $featureFlagName
+     * @param bool $flagState
+     */
+    private function validateConfigurationRowType(string $featureFlagName, bool $flagState): void
+    {
+        // nothing to do, php type validation will reject not valid values
```

Comment:
It makes me think that it's not the first time that I see this method being implemented empty because it HAS to be there since `DataConfigurationInterface` declares it...  kinda annoying